client/src/js/components/bhExchange.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * @overview bhExchange
+ *
+ * @description
+ * This component allows to display the exchange rate list and also
+ * a button for changing the exchange rate
+ */
+angular.module('bhima.components')
+  .component('bhExchange', {
+    templateUrl : 'modules/templates/bhExchange.tmpl.html',
+    controller : bhExchangeController,
+    controllerAs : '$ctrl',
```

Comment:
Just FYI - `.component()` always sets the controller to being `$ctrl` in the view.